modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -27,36 +27,48 @@ The introduction typically provides context for the procedure, such as:
 * Why and where the user performs the procedure
 * Special considerations specific to the procedure
 
+[id="limitations"]
 .Limitations
-This section is optional and not used often. If there are any limitations that the affect the procedure, list them in this section.
+This section is optional and not used often. If there are any limitations that affect procedure, list them in this section. Use a bulleted list for limitations and the plural heading *Limitations*, even if your procedure only has a single limitation.
```

Comment:
I think a second "the" was deleted by mistake - "If there are any limitations that affect **the** procedure, ... "

Reply:
@Jenny-Anne good catch! I fixed it.